src/Impostor.Api/Innersloth/CrossplayFlags.cs

```diff
@@ -0,0 +1,7 @@
+namespace Impostor.Api.Innersloth;
+
+public enum CrossplayFlags
+{
+    Default = 4,
```

Comment:
This may be the default for Steam, but for Itch it is 32. I'm assuming this is following the order in Platforms.cs